front-end/src/components/AboutUsSection/AboutUsSection.jsx

```diff
@@ -0,0 +1,54 @@
+import './AboutUsSection.css';
+import { useState, useEffect } from 'react';
+import { Available } from '../Available';
+import { verticalLogo } from '../../assets/images/logos';
+import { bioData } from '../../mocks/data';
+import { ABOUTUS_TITLE } from '../../constants';
+
+const bioItem = bioData.bio;
+
+export const AboutUsSection = ({ myRef }) => {
+
+    const [isVisible, setIsVisible] = useState(false);
```

Comment:
variable's name should be more meaningful - like elementInViewPort. Then when using it with ternary operator or condition we dont need to compare with true or false value. elementInViewPort or !elementInViewPort with me is more comprehensive than isVisible